File: csharp/isogram/Isogram.cs
using System;
using System.Collections.Generic;

public static class Isogram
{
    public static bool IsIsogram(string word)
    {
        var letters = new List<char>();
        var uniqueLetters = new HashSet<char>();
        foreach (char w in word.ToLower())
        {
            if (Char.IsLetter(w))
            {
                letters.Add(w);
                uniqueLetters.Add(w);
            }
        }
        return letters.Count == uniqueLetters.Count;
    }
}

File: go/isogram/isogram.go
package isogram

import (
	"strings"
	"unicode"
)

func IsIsogram(word string) bool {
	lowerWord := strings.ToLower(word)
	for _, char := range lowerWord {
		if unicode.IsLetter(char) && strings.Count(lowerWord, string(char)) > 1 {
			return false
		}
	}
	return true
}
